Orange/base.py

```diff
@@ -63,15 +63,29 @@ def __call__(self, data):
         model.name = self.name
         model.original_domain = origdomain
         return model
-
+    
     def preprocess(self, data):
         """
         Apply the `preprocessors` to the data.
         """
-        for pp in self.preprocessors:
+        for pp in self.active_preprocessors:
             data = pp(data)
         return data
 
+    @property
+    def active_preprocessors(self):
```

Comment:
What could also work, likely better with fewer symbols to track, is defining this property as simply `preprocessors`. Then its setter stores the user's, and this getter returns the union.